targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/reloc_vector_table.c

```diff
@@ -110,3 +112,11 @@ void nrf_reloc_vector_table(void)
     SCB->VTOR = (uint32_t) nrf_dispatch_vector;    
 #endif
 }
+
+void mbed_sdk_init(void)
+{
+	gpio_t rts;
+	gpio_init_out(&rts, STDIO_UART_RTS);
+	/* Set STDIO_UART_RTS as gpio driven low */
+	gpio_write(&rts, 0);
```

Comment:
This call will trigger an assert for boards with STDIO_UART_RTS set to NC. 

It would be better to wrap everything in an `#if` macro.

Reply:
done